ui/components/MesherySettingsPerformanceComponent.js

```diff
@@ -77,7 +87,7 @@ class MesherySettingsPerformanceComponent extends React.Component {
 
     if (t === '' || !(t.toLowerCase().endsWith('h')
       || t.toLowerCase().endsWith('m') || t.toLowerCase().endsWith('s')) || err || tNum <= 0) {
-      this.setState({ tError: true });
+      this.setState({ tError: 'error-autocomplete-value' });
```

Comment:
@PmcFizz Everything seems awesome, just wanted to know why this change?

Reply:
If the duration  field test don't pass , will add a error class `error-autocomplete-value` to components Autocomplete for show  red error border。 this style of class write on `style/index.css`

Reply:
But that is what tError: true also does I guess

Reply:
If other component maybe tError  is true what is ok. but components ` Autocomplete `  is  not. you can try。

if tError is false，the  border will not change red 。

Reply:
Yes, I was talking in respective of textfield & this is autocomplete! Thanks!